src/test/java/uk/gov/hmcts/darts/common/helper/MediaLinkedCaseHelperTest.java

```diff
@@ -0,0 +1,144 @@
+package uk.gov.hmcts.darts.common.helper;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.darts.common.entity.CourtCaseEntity;
+import uk.gov.hmcts.darts.common.entity.EventEntity;
+import uk.gov.hmcts.darts.common.entity.HearingEntity;
+import uk.gov.hmcts.darts.common.entity.MediaEntity;
+import uk.gov.hmcts.darts.common.entity.MediaLinkedCaseEntity;
+import uk.gov.hmcts.darts.common.entity.UserAccountEntity;
+import uk.gov.hmcts.darts.common.repository.HearingRepository;
+import uk.gov.hmcts.darts.common.repository.MediaLinkedCaseRepository;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.darts.common.enums.MediaLinkedCaseSourceType.AUDIO_LINKING_TASK;
+
+@ExtendWith(MockitoExtension.class)
+class MediaLinkedCaseHelperTest {
+
+    @Mock
+    private MediaLinkedCaseRepository mediaLinkedCaseRepository;
+    @Mock
+    private HearingRepository hearingRepository;
+    @InjectMocks
+    @Spy
+    private MediaLinkedCaseHelper mediaLinkedCaseHelper;
+
+
+    @Test
+    void positiveProcessMedia() {
```

Comment:
I thought we had decided on a naming approach to move forward with 😉 

```suggestion
    void linkMediaByEvent_shouldLinkMediaToCaseAndHearings_whenNotAlreadyLinked() {
```

Reply:
I think we still need to have a call to finalise the naming / testing approach :) 
But in the meantime happy to go with this